app/views/admin/loans/timeline/_batch_actions.html.erb

```diff
@@ -12,14 +12,14 @@
   </li>
 
   <li>
-    <a id="adjust-dates" class="action" data-toggle="modal"
-      data-target="#adjust-dates-modal">
+    <a class="action adjust-dates" data-toggle="modal"
+      data-target="#<%= locals[:modal_id] %>">
```

Comment:
Also please add a comment saying modal_id can be replaced with a hard-coded value when the old timeline goes away.